src/front/glsl/lex.rs

```diff
@@ -302,7 +313,7 @@ impl<'a> Lexer<'a> {
 
     #[must_use]
     pub fn next(&mut self) -> Option<Token> {
-        let (token, rest) = consume_token(&self.input);
+        let (token, rest) = consume_token(&self.input, self.offset == 0);
```

Comment:
is `self.offset` the offset from the line, or the offset from the whole file?

Reply:
It's char offset from start of line, get's reset further down (line 356). Would `char_offset` be a better name?

It's used for calculating position of the token (in TokenMetadata).

Reply:
I think it's typically called a "column" fwiw